pages/en/learn/asynchronous-work/understanding-processnexttick.md

```diff
@@ -47,6 +47,6 @@ process.nextTick(() => {
 ```bash
 Hello => number 1
 Running at next tick => number 2
-Running before the timeout => number 3
 The timeout running last => number 4
+Running before the timeout => number 3
```

Comment:
i have run code on LTS of node and copy paste 😅

Reply:
<img width="702" alt="Capture d’écran 2023-11-21 à 14 42 40" src="https://github.com/nodejs/nodejs.org/assets/97875033/73dc6547-52c3-4fbf-ae6b-0f08e6aa7378">

Reply:
Learn Pages should always reflect the latest state of Node.js -- whenever possible. Hence the learn pages should be updated showing the latest behaviour. We shouldn't even mention the old behavior to avoid confusion.

Reply:
Thanks 👍  Then it seems like the content of this page needs to be updated, the content and the example output say completely different things. I will take a look when I am available 👀